client/me/help/style.scss

```diff
@@ -46,3 +46,38 @@
 	font-size: 14px;
 	color: $gray-dark;
 }
+
+.help__course-teaser .card {
```

Comment:
@omarjackman what is the case where you need to use a different color? We could do `accent="blue"` etc, or `status` to match notices, but I'd like to see where it is needed first.